homeassistant/components/nextdns/sensor.py

```diff
@@ -129,6 +129,17 @@ class NextDnsSensorEntityDescription(
         state_class=SensorStateClass.MEASUREMENT,
         value=lambda data: data.doq_queries,
     ),
+    NextDnsSensorEntityDescription[AnalyticsProtocols](
+        key="tcp_queries",
+        coordinator_type=ATTR_PROTOCOLS,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        entity_registry_enabled_default=False,
+        icon="mdi:dns",
+        name="{profile_name} TCP Queries",
+        native_unit_of_measurement="queries",
+        state_class=SensorStateClass.MEASUREMENT,
```

Comment:
Shouldn't this be a total instead of a measurement?

Reply:
Honestly I'm not sure. Accordind to the documentation, total "state class should not be used for sensors where the absolute value is interesting instead of the accumulated growth or decline". I think the absolute value is interesting for these sensors What is your opinion?

Reply:
The state class impacts what statistics is created.
If state class is "measurement", min, max and mean statistics is created every 5 minutes.
If state class is "total" or "total_increasing", a total sum is updated every 5 minutes.

Reply:
I did a little test, changed the log retention to 1h in the NextDNS configuration and found the API returns values that can go up and down so total_increasing doesn't fit here.

![image](https://user-images.githubusercontent.com/478555/178486554-53203e73-08b5-4dba-b538-d9c1245599b2.png)

I'm confused 🤪 Should I use `total` or `measurement`?

Reply:
That graph is not using long-term statistics and does not show the effect of setting `total_increasing`. Use a statistics cards instead.

Looking at the graph/value, the value seem to be a rolling window?

Reply:
> Looking at the graph/value, the value seem to be a rolling window?

It looks like this is a rolling window and the log retention is taken as the time, unfortunately the API documentation is silent about this.

Reply:
So, what is this value then? 🤷

Reply:
For example, if log retention is set to 1 day the value is the sum o queries from the last 24 hours. This is my assumption, but it fit to my observation. So, you're right, `SensorStateClass.MEASUREMENT` should be used here.

![image](https://user-images.githubusercontent.com/478555/178507886-4bf45fef-5c54-4458-8fee-77c8bac690c3.png)